src/Microsoft.Dnx.Compilation.CSharp/RoslynCompiler.cs

```diff
@@ -286,17 +286,22 @@ private static string RemovePrereleaseTag(string version)
             // Simple reparse of the version string (because we don't want to pull in NuGet stuff
             // here because we're in an old-runtime/new-runtime limbo)
 
-            var dashIdx = version.IndexOf('-');
+            var dashIdx = version?.IndexOf('-');
```

Comment:
Why don't you just check the version for null at the beginning and return null instead of doing this dance with nullables and checking the version twice with the `.?` operator?